presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -681,3 +681,81 @@ The following properties allow tuning the :doc:`/functions/regexp`.
     to hit the limit on matches for subsequent rows as well, you want to use the
     correct algorithm from the beginning so as not to waste time and resources.
     The more rows you are processing, the larger this value should be.
+
+
+Logging Properties
+------------------
+
+Presto uses the server application framework `Airlift
+<https://github.com/airlift/airlift>`_ and therefore supports its configuration
+properties for the HTTP server and general logging.
+
+``log.path``
+^^^^^^^^^^^^
+
+    * **Type:** ``string``
+    * **Default value:** ``var/log/server.log``
+
+    The path to the log file used by Presto. The default is relative to
+    the Presto data directory.
```

Comment:
not sure if a reader will know what data dir is

Reply:
sigh ... I have to port that info from the book as well ... will do in this PR ..

Reply:
done now